Cargo.toml

```diff
@@ -9,21 +9,44 @@ dx12 = ["gfx-backend-dx12"]
 vulkan = ["gfx-backend-vulkan"]
 metal = ["gfx-backend-metal"]
 
-[[bin]]
+[lib]
 name = "ocean"
-path = "src/main.rs"
+path = "src/lib.rs"
+
+[[example]]
```

Comment:
this should be `[[bin]]`, perhaps instead

Reply:
Cargo don't allow to change crate-type for bin target:
```
the target `ocean_ios` is a binary and can't have any crate-types set (currently "staticlib")
```